### DISCUSSION TITLE
Add vega-lite as embedding object

### DISCUSSION THREAD

**Discussion by aborruso** [quarto-dev/quarto-cli#2868] [Feature Requests]
Hi,
it would be great to have in quarto for vega-lite, something like the Mermaid diagrams.

To write something like

````
``` {vega-lite}
{
  "data": {
    "values": [
      {"a": "C", "b": 2}, {"a": "C", "b": 7}, {"a": "C", "b": 4},
      {"a": "D", "b": 1}, {"a": "D", "b": 2}, {"a": "D", "b": 6},
      {"a": "E", "b": 8}, {"a": "E", "b": 4}, {"a": "E", "b": 7}
    ]
  },
  "mark": "point",
  "encoding": {
    "x": {"field": "a", "type": "nominal"}
  }
}
```
````

to get 

![image](https://user-images.githubusercontent.com/30607/195983210-85df1bd6-6fcf-49e6-a2a2-c2bdf659c683.png)

Thank you

**Comment by cscheid** [comment]
Agreed, vega + vega-lite support would indeed be excellent. We'll try to do it in the next 3-6 months.

**Comment by ijlyttle** [comment]
If you are using the knitr engine, the [vegadown](https://vegawidget.github.io/vegadown/articles/vegadown.html) package might work here. 

It was a bit of an experiment, and hasn't made it to CRAN (prob. won't if Vega gets first-class support in Quarto). Maybe it can fit the need in the meantime.

**Reply by aborruso** [reply]
> If you are using the knitr engine, the [vegadown](https://vegawidget.github.io/vegadown/articles/vegadown.html) package might work here.

thank you very much

**Comment by mcanouil** [comment]
Closing in favour of https://github.com/quarto-dev/quarto-cli/issues/2885
